### PR TITLE
Special Immersive Headline

### PR DESCRIPTION
**Pull request by oliverlloyd**
## What does this change?
Set the colours for an immersive headline for special reports

### Before
Screenshot 2021-02-09 at 18 03 58

### After
Screenshot 2021-02-09 at 17 59 55

## Why are we changing types?
When adding new fixture data I pulled in a `GUVideoBlockElement` and the data in the CAPI response for that element contained fields that weren't typed in `CAPIType`, so I've added them here.

### CODE REVIEW

**Comment by oliverlloyd** [comment]
Oh, the special report article I'm using to test with is too long for Chromatic 🤔

File: src/lib/content.d.ts [hunk 1]
```diff
@@ -561,6 +566,13 @@ interface Image {
 interface VideoAssets {
 	url: string;
 	mimeType: string;
+	fields?: {
+		source?: string;
+		embeddable?: string;
+		height?: string;
+		width?: string;
+		caption?: string;
+	};
```
Comment (oliverlloyd) [inline src/lib/content.d.ts:575]: I added these types after refreshing the fixture content and finding them missing

File: src/lib/content.d.ts [hunk 2]
```diff
@@ -552,6 +555,8 @@ interface Image {
 		height: string;
 		width: string;
 		isMaster?: string;
+		source?: string;
+		caption?: string;
```
Comment (oliverlloyd) [inline src/lib/content.d.ts:559]: I added these types after refreshing the fixture content and finding them missing

File: src/lib/content.d.ts [hunk 3]
```diff
@@ -163,6 +163,9 @@ interface GuVideoBlockElement {
 	html: string;
 	source: string;
 	role?: RoleType;
+	imageMedia?: { allImages: Image[] };
+	originalUrl?: string;
+	url?: string;
```
Comment (oliverlloyd) [inline src/lib/content.d.ts:168]: I added these types after refreshing the fixture content and finding them missing

**Comment by oliverlloyd** [comment]
[Special Immersive Articles](https://trello.com/c/nwZ3B1KS/2440-special-immersive-articles)

**Review by liywjl — APPROVED** [review]
👍

**Pull request merged**